server/config/toml.go

```diff
@@ -21,6 +21,10 @@ const DefaultConfigTemplate = `# This is a TOML config file.
 # specified in this config (e.g. 0.25token1;0.0001token2).
 minimum-gas-prices = "{{ .BaseConfig.MinGasPrices }}"
 
+# The maximum gas a query coming over rest/grpc may consume.
```

Comment:
Can you add this here: https://github.com/cosmos/cosmos-sdk/blob/b590b091076078c7e5c740e9c9f3f2f98f7e85e7/tools/confix/data/v0.48-app.toml#L1 (btw if you can rename it to v0.50 that'd be great 😬)